sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/__init__.py

```diff
@@ -110,6 +123,23 @@
     "AzureOpenAIStringCheckGrader",
     "AzureOpenAITextSimilarityGrader",
     "AzureOpenAIScoreModelGrader",
+    # Include lazy imports in __all__ so they appear as available
+    "AIAgentConverter",
+    "SKAgentConverter",
 ]
 
 __all__.extend([p for p in _patch_all if p not in __all__])
+
+
+def __getattr__(name):
+    """Handle lazy imports for optional dependencies."""
+    if name in _lazy_imports:
+        try:
+            return _lazy_imports[name]()
```

Comment:
Do return directly and let the import error be raise do not catch and convert it